### PR TITLE
Fix issue #50

### PR DESCRIPTION
**Pull request by sajithaliyanage**
- Closes https://github.com/cibr-qcri/airflow-tasks/issues/50

### CODE REVIEW

File: scripts/eth-contract-security-score/security_score_calculator.py [hunk 1]
```diff
@@ -70,33 +70,53 @@ def persist_riskscore(data_processed):
         print("Error while updating to PostgreSQL", error)
 
 
-def calculate_riskscore_and_store():
-    riskscore_address_data = list()
+def calculate_security_score_and_store():
```
Comment (thedarkspy) [inline scripts/eth-contract-security-score/security_score_calculator.py:73]: calulate_and_store_security_score

File: scripts/eth-contract-security-score/security_score_calculator.py [hunk 2]
```diff
@@ -70,33 +70,53 @@ def persist_riskscore(data_processed):
...
+    securityscore_address_data = list()
```
Comment (thedarkspy) [inline scripts/eth-contract-security-score/security_score_calculator.py:74]: security_score

File: scripts/eth-contract-security-score/security_score_calculator.py [hunk 3]
```diff
@@ -123,7 +143,9 @@ def main():
             severities[severity] = 1
             address_vulnerability_mapper[address] = severities
 
-    calculate_riskscore_and_store()
+    calculate_security_score_and_store()
+
```
Comment (thedarkspy) [inline scripts/eth-contract-security-score/security_score_calculator.py:147]: change here also

**Review by thedarkspy — CHANGES_REQUESTED** [review]

File: scripts/eth-contract-security-score/security_score_calculator.py (hunk 1 above)
Comment (sajithaliyanage) [reply scripts/eth-contract-security-score/security_score_calculator.py]: Done

File: scripts/eth-contract-security-score/security_score_calculator.py (hunk 2 above)
Comment (sajithaliyanage) [reply scripts/eth-contract-security-score/security_score_calculator.py]: Done

File: scripts/eth-contract-security-score/security_score_calculator.py (hunk 3 above)
Comment (sajithaliyanage) [reply scripts/eth-contract-security-score/security_score_calculator.py]: Done

**Review by thedarkspy — APPROVED** [review]

**Review by thedarkspy — APPROVED** [review]

**Pull request merged**